tests/fsharpqa/Source/Diagnostics/NONTERM/tuplewithlazy01b.fs

```diff
@@ -1,8 +1,8 @@
 // #Regression #Diagnostics 
 // Regression test for DevDiv:64339
 // Note that the bug still repros in CHK/DBG bits - we will knownfail it
-//<Expects status="error" span="(8,25-8,29)" id="FS0010">Unexpected keyword 'lazy' in type definition$</Expects>
-//<Expects status="error" span="(11,29-11,33)" id="FS0010">Unexpected keyword 'lazy'$</Expects>
+//<Expects status="error" span="(8,25-8,29)" id="FS0010">Unexpected keyword 'lazy' in pattern$</Expects>
```

Comment:
"Pattern" is a bit less precise than "type definition". Is this a necessary sacrifice for this type of parser simplification?

Reply:
It's actually more precise, because the parameter pattern is where the error happened, not the outer type declaration. Even if it was worse, I'd say it'd worth it, because it's more about making parsing more robust than about simplifying it.

Reply:
Alright, thanks for the explanation!